src/libraries/Microsoft.Extensions.Logging.Abstractions/ref/Microsoft.Extensions.Logging.Abstractions.cs

```diff
@@ -126,6 +126,19 @@ public enum LogLevel
 }
 namespace Microsoft.Extensions.Logging.Abstractions
 {
+    public readonly partial struct LogEntry<TState>
+    {
+        private readonly TState _State_k__BackingField;
```

Comment:
>_State_k__BackingField [](start = 32, length = 22)

are all private fields really needed? isn't the _dumy one is enough?

Reply:
I think this due to the generic parameter.

Reply:
yes. this is autogenerated by GenerateReferenceSource